src/words.ts

```diff
@@ -43,7 +43,7 @@ export const words = async (message: Message) => {
 // Utility function for removing special characters
 function stripSpecialCharacters(str: string) {
   // match special symbols and replace with ' '
-  str = str.replace(/[.,/#!$%?&*;:{}=\-_'"~()]/g, ' ');
+  str = str.replace(/[.,/#!$%?&*;:{}=\-_'"“”~()]/g, ' ');
```

Comment:
LGTM-!
btw, wouldn't the regex below work similarly? 
```suggestion
  str = str.replace(/[.,/#!$%?&*;:{}=\-_'"”~()]/g, ' ');
```